signer/plugin.go

```diff
@@ -44,6 +45,10 @@ func NewFromPlugin(plugin plugin.Plugin, keyID string, pluginConfig map[string]s
 	}, nil
 }
 
+func (s *pluginSigner) PluginAnnotations() map[string]string {
```

Comment:
We need a func doc for this function and mention that `pluginSigner implements notation.signerAnnotation`.